kafka_utils/kafka_check/commands/under_replicated.py

```diff
@@ -41,17 +41,34 @@ def run_command(self):
             REPLICA_NOT_AVAILABLE_ERROR,
         )
 
-        if not under_replicated:
-            return status_code.OK, 'No under replicated partitions.'
+        errcode = status_code.OK if not under_replicated else status_code.CRITICAL
+        out = _prepare_output(under_replicated, self.args.json, self.args.verbose)
+        return errcode, out
+
+
+def _prepare_output(partitions, json, verbose):
```

Comment:
I personally don't like that this function returns wither a dictionary or a string. I would make it return always a dictionary and convert it either into string or json in terminate().